src/test/resources/doc/examplesExpected/configChildren.xsd

```diff
@@ -0,0 +1,402 @@
+          <xs:attribute name="className" type="xs:string" use="required" />
+          <xs:anyAttribute processContents="skip" />
+        </xs:complexType>
+      </xs:element>
+      <xs:group ref="AElementGroupBase_2" />
+    </xs:choice>
+  </xs:group>
+  <xs:group name="BElementGroup_2">
+    <xs:choice>
+      <xs:element name="B">
+        <xs:complexType>
+          <xs:attribute ref="active" />
+          <xs:attribute name="elementRole" type="xs:string" fixed="b" use="prohibited" />
+          <xs:attribute name="className" type="xs:string" use="required" />
+          <xs:anyAttribute processContents="skip" />
+        </xs:complexType>
+      </xs:element>
+      <xs:group ref="BElementGroupBase" />
+    </xs:choice>
+  </xs:group>
+  <xs:complexType name="Level2Type">
+    <xs:group ref="Level2CumulativeChildGroup" />
+    <xs:attribute ref="active" />
+    <xs:attribute name="className" type="xs:string" fixed="org.frankframework.frankdoc.testtarget.examples.config.children.Level2" use="prohibited" />
+  </xs:complexType>
+  <xs:group name="Level2DeclaredChildGroup">
+    <xs:sequence>
+      <xs:group ref="CElementGroup" minOccurs="0" maxOccurs="unbounded" />
+    </xs:sequence>
+  </xs:group>
+  <xs:group name="Level2CumulativeChildGroup">
+    <xs:sequence>
+      <xs:group ref="Level2DeclaredChildGroup" />
+      <xs:group ref="Level1DeclaredChildGroup" />
+    </xs:sequence>
+  </xs:group>
+  <xs:complexType name="Level3Type">
+    <xs:group ref="Level3CumulativeChildGroup" />
+    <xs:attribute ref="active" />
+    <xs:attribute name="className" type="xs:string" fixed="org.frankframework.frankdoc.testtarget.examples.config.children.Level3" use="prohibited" />
+  </xs:complexType>
+  <xs:group name="Level3DeclaredChildGroup">
+    <xs:sequence>
+      <xs:group ref="AElementGroup_3" minOccurs="0" maxOccurs="unbounded" />
+      <xs:group ref="DElementGroup" minOccurs="0" maxOccurs="unbounded" />
+    </xs:sequence>
+  </xs:group>
+  <xs:group name="Level3CumulativeChildGroup">
+    <xs:sequence>
+      <xs:group ref="Level3DeclaredChildGroup" />
+      <xs:group ref="Level2DeclaredChildGroup" />
+      <xs:group ref="BElementGroup_2" minOccurs="0" maxOccurs="unbounded" />
+    </xs:sequence>
+  </xs:group>
+  <xs:complexType name="LevelWithCumulativeType">
+    <xs:group ref="LevelWithCumulativePluralConfigChildGroup" />
+    <xs:attribute ref="active" />
+    <xs:attribute name="className" type="xs:string" fixed="org.frankframework.frankdoc.testtarget.examples.config.children.LevelWithCumulative" use="prohibited" />
+  </xs:complexType>
+  <xs:group name="LevelWithCumulativePluralConfigChildGroup">
+    <xs:sequence>
+      <xs:choice minOccurs="0" maxOccurs="unbounded">
+        <xs:group ref="AElementGroup_2" />
+        <xs:group ref="BElementGroup_2" />
+        <xs:group ref="CElementGroup" />
+        <xs:group ref="DElementGroup" />
+      </xs:choice>
```

Comment:
When a parent class has plural config children, it can be inherited by a descendant that introduces new config children. This case probably does not occur in practice, so we did not add intelligence for it. The result should be valid though. I am satisfied that the descendant lists all declared and inherited config children.